content/documentation/github.md

```diff
@@ -1,101 +1,143 @@
+{{< q-figure id="github-desktop-publish" >}}
 
-Think about your commits in meaningful bundles, that could be undone without effecting other areas of your project. Rather than committing all the random changes you made to a single file, or everything you did before lunch that day, commit changes that make sense together.
+Your project code is now on GitHub! To see it, go to Repository > View on GitHub or visit https://github.com/YOUR-USERNAME/YOUR-PROJECT-NAME.
 
-{{< q-class "box tip" >}}
+[[Insert image of a repository?]]
 
-- You can break changes into multiple commits by using the check boxes in the lefthand list to select which files will or won’t be included in a given commit.
+## Clone an Existing Project from GitHub
 
+If you are collaborating on a project, and one of your colleagues has already hosted your project code on GitHub, you can clone (or make a copy) of this repository to work on it locally.
+
+1. Click on the upper left-hand corner of GitHub Desktop where it says "Current Repository." Click "Add" and choose the option to "Clone repository..."
+
+2. Click the URL menu option and cut-and-paste the URL (https://github.com/USERNAME/PROJECT-NAME) for your desired repository. Be sure to choose the local path where you want your project to live on your computer. (We recommend creating a GitHub folder in your home directory and storing the project in there.)
+
+Alternately, you can copy this project directly from GitHub.com. Navigate to the project on GitHub, click the green “Code” button, select “Open with GitHub Desktop,” and follow the prompts.
+
+{{< q-class "box warning" >}}
+- To view a preview of a Quire project cloned from GitHub, you’ll need to first run `quire install` in your project directory. Then run `quire preview` and navigate to [http://localhost:1313/](http://localhost:1313/).
 {{< /q-class >}}
 
-Even after saving your changes in your text editor and committing them in GitHub Desktop, GitHub itself still doesn’t know about them. For that you need to “push” your commits to GitHub. For that simply use the "Push origin" link in the upper right. Pushing can include one or many commits, and you can push as often as you’d like, but it’s typically good to do it at least a couple times day when actively working on a project.
+You now have a copy of that project on your computer that is still connected to the original code on GitHub. You are ready to collaborate!
 
-## Using GitHub to Collaborate with Others on Your Project
+[[Mention forking somewhere. Explanations about forking: https://github.community/t/the-difference-between-forking-and-cloning-a-repository/10189 and https://opensource.com/article/17/12/fork-clone-difference and https://thenewstack.io/getting-legit-with-git-and-github-cloning-and-forking/]]
```

Comment:
I'm not sure about forking, but I think you do want to mention that to collaborate on a colleague's existing project, that colleague should add them to the project as a collaborator, or the project should be hosted in an organizational GtiHub account where everyone is a member of that organization. (Which I see you explain in the Collaborate with Others section, so maybe just copy that nit up here as well?) Forking would be for more of an outside contributor who is more speculatively making changes to a project repository, rather than for someone on the project team, which is I think how how you're framing it here.